src/test/java/com/pajehyk/files/FilesTest.java

```diff
@@ -0,0 +1,31 @@
+package com.pajehyk.files;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Test;
+
+public class FilesTest {
+    @Test 
+    public void runTest() {
+        String expectedFilePath = "src/test/resources/expectedOutput.txt";
+        String actualFilePath = "src/test/resources/actualOutput.txt";
+        String[] args = {"src/test/java/com/pajehyk/files", "src/test/resources/actualOutput.txt"};
```

Comment:
Please create predefined file strucure in test/resources and use it to check the code